common/tools/dev-tool/test/resolveProject.spec.ts

```diff
@@ -19,13 +19,12 @@ describe("Project Resolution", async () => {
     await assert.isRejected(resolveProject(p), /filesystem root/);
   });
 
-  // Issue: https://github.com/Azure/azure-sdk-for-js/issues/13202
-  it.skip("resolution finds dev-tool package", async () => {
+  it("resolution finds dev-tool package", async () => {
     const packageInfo = await resolveProject(__dirname);
     assert.equal(packageInfo.name, "@azure/dev-tool");
     assert.match(
       packageInfo.path,
-      new RegExp(`.*${path.sep}${path.join("common", "tools", "dev-tool")}`)
+      new RegExp(`.*${(path.sep + path.join("common", "tools", "dev-tool")).replace(/\\/g, "\\\\")}`)
```

Comment:
I do not understand the purpose of the replacement here

Reply:
I'll add a comment here to explain, but it's because the RegExp compiler treats the single backslash as an escape signifier, so it needs to be escaped _again_. This would have been an error during regexp compilation if not for the bewildering fact that \co (from \common), \t (from \tools), and \d (from \dev-tool) are all valid escape strings.